locales/es.json

```diff
@@ -2087,7 +2086,7 @@
 			}
 		},
 		"button": {
-			"archive": "Archivar",
+			"archive": "Termina",
```

Comment:
Should probably be "Terminar" to make it the Infinitiv like we usually use it, this is the Befehlsform I think. Not 100% sure though